src/toolbox/toolbox_operations.py

```diff
@@ -25,21 +25,37 @@
 """
 
 import unicodedata
-from typing import List
+from typing import Any, Dict, List
 
 
 def toolbox_file_reader(filename):  # read in toolbox file
-    with open(filename, "r") as f:
+    with open(filename, "r", encoding="iso8859_5") as f:
```

Comment:
Add a comment:
"Use the iso8859_5 encoding to avoid problems with characters which are not utf-8"